### ISSUE TITLE
Income Limits Automation: Create a Github usable workflow in vets-api

### ISSUE THREAD

**Issue by dsasser** [department-of-veterans-affairs/va.gov-cms#15203]
## Description
Create a Github Workflow that executes inside a self-hosted runner, to test access to VES.

We are developing a Github Workflow which will eventually export data from VES to CSV. The workflow will upload the CSVs to S3 to be consumed by existing vets-api Sidekiq import scripts. However, before we can do that, we need to validate that the Workflow functions like it did in the va.gov-cms repo [when we executed a test Workflow there](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15202).

### User story
**AS AN** Income limits owner 
**I WANT** the app to test connectivity to the VES database
**SO THAT** we are confident that we can proceed with automating a data import.

## Acceptance criteria
- [ ] Github workflow exists in vets-api
- [ ] Workflow executes with a meaningful output that validates that it's working
- [ ] Create new ticket for: running the queries and writing to S3

**Comment by jilladams** [comment]
Added an AC around monitoring. We'll need to think about how we get told / know if the export fails.

**Comment by jilladams** [comment]
I also noticed this ticket is nicely agnostic about whether / how we get this done, whether via Github Actions or otherwise. If we do go with Github Actions, we have some existing examples via content-build re: how Slack notifications can be used for failures. e.g. 

* Failed content-release builds: https://github.com/department-of-veterans-affairs/content-build/actions/workflows/content-release.yml?query=is%3Afailure
* [#status-content-build](https://dsva.slack.com/archives/C02VD909V08) channel tracks status of the builds
*  Datadog monitor for time since last successful release: 
  * Slack example: https://dsva.slack.com/archives/CDHBKAL9W/p1692723062674929
  * Actual Datadog monitor: https://vagov.ddog-gov.com/monitors/157216 
    * With read-only DataDog permissions, I do not have perms to create a new Monitor, so this process will likely require a CMS Team devops help, or an increased DataDog permission in order to create a new monitor. 
    * This monitor is tied to a PagerDuty rotation, which, afaik, is only available to Platform team members. We shouldn't be planning to do that part.

**Comment by FranECross** [comment]
Won't be complete on the first pass; will need to get Platform involved to get permissions set up.

**Comment by dsasser** [comment]
## Status Update 12/1/23

The [Workflow has been created](https://github.com/department-of-veterans-affairs/vets-api/pull/14718), and the test was successful:

<img width="460" alt="Screenshot 2023-12-01 at 8 47 19 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/221539/6ea59dbd-b714-4f54-8a3a-c191c757a714">

We can now proceed with the [next ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16217) which will create the initial query and CSV exports.

**Comment by dsasser** [comment]
Closing, as test was successful.

**Comment by dsasser** [comment]
@FranECross @jilladams 

Regarding this AC:

> Github workflow exists in vets-api

Since the workflow test contains sensitive info, I opted to destroy the PR (after cleaning up the references to the connection test details). I created a Workflow file starter template in the next ticket so we can pick up where we left off, but don't risk any data leaking into the public by keeping the existing test workflow in place.
